<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Register</class>
 <widget class="QWidget" name="Form_Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pedollc.qrc">
    <normaloff>:/PeDollc/Resources/key.png</normaloff>:/PeDollc/Resources/key.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mac IDï¼š</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_MacID">
         <property name="text">
          <string>FFFFFF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Key">
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Buy">
       <property name="text">
        <string>Buy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pedollc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Form_Register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
